<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="vlastnosti">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>vlastnosti</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>šířka čáry</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="carSir">
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>čára G</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>čára B</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>čára R</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="carR">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="carG">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>čára A</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="carB">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="carA">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>výplň R</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>výplň G</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>výplň B</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>výplň A</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="fillR">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="fillG">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="fillB">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="fillA">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="barvaCanvas">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>barva canvas</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="G">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="B">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="R">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="A">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVLastnosti">
    <property name="title">
     <string>Vlastnosti</string>
    </property>
    <addaction name="actionZobraz"/>
   </widget>
   <widget class="QMenu" name="menuKresli">
    <property name="title">
     <string>Kresli...</string>
    </property>
    <widget class="QMenu" name="menuN_uhelnik_nepravidelny">
     <property name="title">
      <string>N úhelník nepravidelný</string>
     </property>
     <addaction name="action3"/>
     <addaction name="action4"/>
     <addaction name="action5"/>
     <addaction name="action6"/>
     <addaction name="action7"/>
     <addaction name="action8"/>
     <addaction name="action9"/>
     <addaction name="action10"/>
     <addaction name="action11"/>
     <addaction name="action12"/>
    </widget>
    <widget class="QMenu" name="menuN_heln_k_pravideln">
     <property name="title">
      <string>N úhelník pravidelný</string>
     </property>
     <addaction name="action3_2"/>
     <addaction name="action4_2"/>
     <addaction name="action5_2"/>
     <addaction name="action6_2"/>
     <addaction name="action7_2"/>
     <addaction name="action8_2"/>
     <addaction name="action9_2"/>
     <addaction name="action10_2"/>
     <addaction name="action11_2"/>
     <addaction name="action12_2"/>
    </widget>
    <widget class="QMenu" name="menuN_heln_k_prav_ze_st_edu">
     <property name="title">
      <string>N úhelník prav. ze středu</string>
     </property>
     <addaction name="action3_3"/>
     <addaction name="action4_3"/>
     <addaction name="action5_3"/>
     <addaction name="action6_3"/>
     <addaction name="action7_3"/>
     <addaction name="action8_3"/>
     <addaction name="action9_3"/>
     <addaction name="action10_3"/>
     <addaction name="action11_3"/>
     <addaction name="action12_3"/>
    </widget>
    <addaction name="actionUsecku"/>
    <addaction name="actionKruh"/>
    <addaction name="menuN_uhelnik_nepravidelny"/>
    <addaction name="menuN_heln_k_pravideln"/>
    <addaction name="menuN_heln_k_prav_ze_st_edu"/>
    <addaction name="actionObd_ln_k"/>
    <addaction name="actionElipsu"/>
   </widget>
   <widget class="QMenu" name="menuCanvas">
    <property name="title">
     <string>canvas</string>
    </property>
    <widget class="QMenu" name="menuAntialias">
     <property name="title">
      <string>antialias</string>
     </property>
     <addaction name="actionZapnout"/>
     <addaction name="actionVypnout"/>
    </widget>
    <addaction name="menuAntialias"/>
    <addaction name="actionBarva"/>
   </widget>
   <addaction name="menuVLastnosti"/>
   <addaction name="menuKresli"/>
   <addaction name="menuCanvas"/>
  </widget>
  <action name="actionZobraz">
   <property name="text">
    <string>Zobraz</string>
   </property>
  </action>
  <action name="actionUsecku">
   <property name="text">
    <string>Úsečku</string>
   </property>
  </action>
  <action name="actionKruh">
   <property name="text">
    <string>Kruh</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action11">
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action6_2">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7_2">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8_2">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9_2">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action11_2">
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="action12_2">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action10_2">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action6_3">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action5_3">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action4_3">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action3_3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action7_3">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8_3">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9_3">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action10_3">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action11_3">
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="action12_3">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="actionZapnout">
   <property name="text">
    <string>zapnout</string>
   </property>
  </action>
  <action name="actionVypnout">
   <property name="text">
    <string>vypnout</string>
   </property>
  </action>
  <action name="actionBarva">
   <property name="text">
    <string>barva</string>
   </property>
  </action>
  <action name="actionObd_ln_k">
   <property name="text">
    <string>Obdélník</string>
   </property>
  </action>
  <action name="actionElipsu">
   <property name="text">
    <string>Elipsu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
